Util/Xamarin.Kotlin.BindingSupport/native/src/main/kotlin/xamarin/binding/kotlin/bindingsupport/ProcessXmlCommand.kt

```diff
@@ -20,6 +20,9 @@ class ProcessXmlCommand : CliktCommand(name = "KotlinBindingSupport") {
     val outputFile: File? by option("-o", "--output", help = "path to the output transform file")
         .file(fileOkay = true, folderOkay = false)
 
+    val ignoreFile: File? by option("-i", "--ignore", help = "path to the ignore file")
+        .file(fileOkay = true, folderOkay = false)
```

Comment:
I don't actually thing this is needed anymore since it is working properly with a lower Kotlin version, but just in case.